src/tracer.test.js

```diff
@@ -39,6 +37,8 @@ describe('tracer', () => {
   spies.clearGlobals = jest.spyOn(globals, 'clearGlobals');
   spies.warnClient = jest.spyOn(logger, 'warnClient');
   spies.logFatal = jest.spyOn(logger, 'fatal');
+  spies.log = jest.spyOn(console, 'log');
```

Comment:
Why did you add these spies?

Reply:
Now he dont print logs when you run npm run test